_pages/how-we-work/tools/google-meet.md

```diff
@@ -42,6 +42,16 @@ If the audio for Hangouts has stopped working, you can either reset your compute
 4. If it asks for your password, type your computer log-in password and then hit enter
 5. Audio should now work, but if it doesn't, try re-starting your computer
 
+### Presenting a deck in hangouts
+If you're doing a virtual presentation it can be a bit tricky, so here are some tips to both see what you are presenting and who you are presenting to.
+
+1. Open up in your browser your slides presentation. (Window a)
```

Comment:
The split infinitive here makes this tricky to read. I'd recommend either "Open your Google Slides presentation in a browser window" or "In your browser, open your Slides presentation."